lib/cisco_node_utils/bgp.rb

```diff
@@ -524,6 +582,7 @@ def reconnect_interval=(seconds)
     end
 
     def default_reconnect_interval
+      return nil if platform == :ios_xr
```

Comment:
Doesn't config_get_default already return nil if the yaml doesn't specify XR config?